automation_tools/__init__.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import print_function
 import os
 import random
+import requests
 import socket
 import sys
 import time
 import novaclient
 import subprocess
+from lxml import html
```

Comment:
@elyezer ok. Anyway, xml.Etree has trouble parsing the document. Looks like it is malformed in its eyes. I vote for staying with lxml.